examples/2d/rect.rs

```diff
@@ -9,6 +11,8 @@ fn main() {
 
 fn setup(mut commands: Commands) {
     commands.spawn_bundle(OrthographicCameraBundle::new_2d());
+
+    // Since this Sprite is not generated from an image, we have to specify its size in `custom_size`.
```

Comment:
```suggestion
    // Since this `Sprite` is not generated from an image, we have to specify its size in `custom_size`.
```

Monospace consistency.